mappings/net/minecraft/block/Block.mapping

```diff
@@ -69,6 +69,15 @@ CLASS net/minecraft/class_2248 net/minecraft/block/Block
 	METHOD method_35258 dropStacks (Lnet/minecraft/class_2680;Lnet/minecraft/class_47$class_48;)V
 		ARG 0 state
 		ARG 1 lootContext
+	METHOD method_36992 dropStack (Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;Lnet/minecraft/class_2350;Lnet/minecraft/class_1799;)V
+		ARG 0 world
+		ARG 1 pos
+		ARG 2 direction
+		ARG 3 stack
+	METHOD method_36993 dropStack (Lnet/minecraft/class_1937;Ljava/util/function/Supplier;Lnet/minecraft/class_1799;)V
+		ARG 0 world
+		ARG 1 itemEntityGetter
```

Comment:
```suggestion
		ARG 1 itemEntitySupplier
```

Reply:
Agree with haykam. This is to avoid extra item creation in case the drop shouldn't be spawned than a getter that retrieves an existing item.